tests/utilities_t/test_asynccachemap.py

```diff
@@ -55,12 +55,12 @@ def test_iter_async_cache_map(self):
             self.assertEqual(value, self.test_data.get(key))
 
     def test_json_failing_update(self):
-        with self.assertLogs(logging.getLogger(), logging.ERROR):
+        with self.assertLogs(logging.getLogger(), logging.WARNING):
```

Comment:
```suggestion
        with self.assertLogs(level=logging.WARNING):
```

Reply:
Explicitly using the root-level logger logging.getLogger() is equivalent to the default behavior of assertLogs.